tests/MoneyTest.php

```diff
@@ -26,4 +27,13 @@ public function testJsonEncoding()
             json_encode(Money::EUR(350))
         );
     }
+
+    public function testMaxInit()
```

Comment:
@sagikazarmark I was doing that, was fighting with it actually because I got errors that I was not able to fix quickly. Decided to leave it. My bad.